setup.py

```diff
@@ -0,0 +1,60 @@
+from setuptools import setup, find_packages
+
+requirements = [
+    'aiosignal',
+    'attrs',
+    'autogluon.common',
+    'boto3',
+    'botocore',
+    'certifi',
+    'charset-normalizer',
+    'click',
+    'contourpy',
+    'cycler',
+    'exceptiongroup',
+    'filelock',
+    'fonttools',
+    'frozenlist',
+    'idna',
+    'iniconfig',
+    'jmespath',
+    'jsonschema',
+    'jsonschema-specifications',
+    'kiwisolver',
+    'matplotlib',
+    'msgpack',
+    'numpy',
+    'packaging',
+    'pandas',
+    'pillow',
+    'pluggy',
+    'protobuf',
+    'psutil',
+    'pyparsing',
+    'pytest',
+    'python-dateutil',
+    'pytz',
+    'PyYAML',
+    'ray',
+    'referencing',
+    'requests',
+    'rpds-py',
+    's3transfer',
+    'scipy',
+    'seaborn',
+    'six',
+    'tomli',
+    'tqdm',
+    'tzdata',
+    'urllib3',
+]
```

Comment:
Avoid duplicate code. Either have the dependencies here or in `requirements.txt`, not both

Reply:
Better to clean this and only include the dependencies explicitly imported in the package, rather than dependencies of dependencies